### PR TITLE
Color contrast improvements

### PR DESCRIPTION
**Pull request by philipqueen**
This pull request solves a number of the major color contrast issues in the GUI, although not all of them.

First, it fixes the major mac darkmode problem by setting a white background on all of the QLineEdit objects, turning this: 
Screen Shot 2023-04-04 at 12 50 28 PM
into this:
Screen Shot 2023-04-04 at 3 32 10 PM

Second, it increases the color contrast of certain text, like the parameter tree text is now dark instead of white:
Screen Shot 2023-04-04 at 3 33 54 PM
And the text for the link to the privacy policy is now dark instead of white:
Screen Shot 2023-04-04 at 3 34 29 PM

The only remaining issues are the user selections in the parameter tree that are still displaying dark on a mac in darkmode. They are only dark when they are selected (you can see them just fine when not selected). I can't figure out how to fix them in the css. Examples of what I'm talking about:

Screen Shot 2023-04-04 at 3 03 46 PM
Screen Shot 2023-04-04 at 3 03 58 PM

### CODE REVIEW

File: freemocap/gui/qt/widgets/camera_controller_group_box.py [hunk 1]
```diff
@@ -65,7 +67,13 @@ def _create_videos_will_save_to_layout(self):
         self._recording_path_label = QLabel(f"{self.get_new_recording_path()}")
         self._recording_path_label.setStyleSheet("font-family: monospace; font-size: 10px;")
         hbox.addWidget(self._recording_path_label)
-        return hbox
+
+        vbox.addLayout(hbox)
+
+        lag_note_label = QLabel("NOTE: Use fewer cameras or decrease resolution if you are experiencing lag")
```
Comment (jonmatthis) [inline freemocap/gui/qt/widgets/camera_controller_group_box.py:73]:
```suggestion
        lag_note_label = QLabel("NOTE: If you experience lag in your camera views, decrease the resolution and/or use fewer cameras.\nWe are working on a fix which should be done 'soon' (written: 2023-04-05)")
```

Changed my mind again on this text, please make sure that newline `\n` works properly in the GUI

**Review by jonmatthis — CHANGES_REQUESTED** [review]
we gotta fix that "black text on black bkgrd" thing before we merge - we can pair on it

**Comment by philipqueen** [comment]
The new line renders properly in the note text. It might look nicer if we center it but thats ok

Screen Shot 2023-04-05 at 9 36 34 AM

**Comment by philipqueen** [comment]
@jonmatthis This is all ready except for some lingering black on black inside the parameter trees (for processing and camera settings). I've had no luck fixing it myself, if you'd like to pair on it.

**Review by jonmatthis — APPROVED** [review]

**Pull request merged**